includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -53,15 +53,16 @@ public function __construct( $gateway ) {
 	 * @return string
 	 */
 	public function get_request_url( $order, $sandbox = false ) {
+		if ( $sandbox ) {
+			$this->request_url_prefix = 'https://www.sandbox.paypal.com/cgi-bin/webscr?test_ipn=1&';
```

Comment:
Just `endpoint` may be cleaner naming for this local param. Can we define it protected also?